core-sitemaps.php

```diff
@@ -2,13 +2,13 @@
 /**
  * Main setup.
  *
- * @package         Core_Sitemaps
+ * @package Sitemaps
  */
 
 /**
  * Core Sitemaps Plugin.
  *
- * @package   Core_Sitemaps
+ * @package   Sitemaps
```

Comment:
Not a big deal, but I feel it's safer if we use either `Core_Sitemaps` or `WP_Sitemaps` everywhere, but not the plain `Sitemaps`, since this will be more error-prone to replace later. Since this file will become irrelevant, it's okay here, but wanted to point it out so that we're careful about this.

Reply:
Fixed now to avoid confusion.

Reply:
`@subpackage Sitemaps` seems fine to me?